tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenDatabaseTest.cs

```diff
@@ -15,49 +15,27 @@
 
 using System.Collections.Generic;
 using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
 
 namespace MongoDB.Driver.Tests.JsonDrivenTests
 {
-    public abstract class JsonDrivenDatabaseTest : JsonDrivenClientTest
+    public abstract class JsonDrivenDatabaseTest : JsonDrivenCommandTest
```

Comment:
This is the base class for all tests where _database is the receiver of the method call.